app/assets/javascripts/app/components/comment_form/comment_form_ctrl.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+
+  angular
+  .module('council.components')
+  .controller('CommentFormCtrl', CommentFormCtrl);
+
+  function CommentFormCtrl(DS) {
+    var ctrl = this;
+
+    ctrl.resetComment = resetComment;
+    ctrl.updateComment = updateComment;
```

Comment:
I guess having the suffix comment in this controller (which belongs to the comment directive and is used in the comment view, basically a comment component) is kind of redundant